app/views/catarse_bootstrap/projects/_dashboard_posts.html.slim

```diff
@@ -1,16 +1,16 @@
-.section
+.w-section.section
   .w-container
     .w-row
-      .w-col.w-col-8
-        .project-news.u-marginbottom-60
-          .fontsize-large.fontweight-semibold.u-marginbottom-30= t('.title')
-          = simple_form_for @project,  html: { class: 'w-form card card-terciary' } do |f|
+      .w-col.w-col-1
```

Comment:
I think we should remove this empty w-col-1 div and add .w-col-push-1 to the colune where the content is.